donate/payments/forms.py

```diff
@@ -13,14 +13,18 @@
 from . import constants
 from .utils import get_currency_info
 
+from ..settings.environment import root
+
 import json
 
-# # Loading a JSON list of countries and their respective post code formats if applicable,
-# # from the source/js directory.
+# Loading a JSON list of countries and their respective post code formats if applicable,
+# from the source/js directory.
 try:
-    with open('./source/js/components/post-codes-list.json') as post_code_data:
+    with open(f'{root}/source/js/components/post-codes-list.json') as post_code_data:
         COUNTRY_POST_CODES = json.load(post_code_data)
-except Exception:
+except Exception as error:
+    print('ERROR: could not read in post codes list')
+    print(error)
```

Comment:
Let's switch this to using the `logger` like in https://github.com/mozilla/donate-wagtail/blob/master/donate/payments/tasks.py ?

Reply:
Done! Marking as resolved :D